tests/integration/services/community.test.ts

```diff
@@ -1259,4 +1262,66 @@ describe('community service', () => {
             expect(result.email).to.be.equal('');
         });
     });
+
+    describe('get manager', () => {
+        afterEach(async () => {
+            await truncate(sequelize, 'Manager');
+            await truncate(sequelize, 'Beneficiary');
+            await truncate(sequelize, 'Community');
+            await truncate(sequelize);
+        });
+
+        it('should return a list of managers', async () => {
```

Comment:
We need more tests.
I suggest renaming this one to `should return a list of added beneficiaries by current managers`
And also add
`should return a list of added beneficiaries by previous managers (deleted accounts)`
`should return a list of added beneficiaries by previous managers (existing accounts but not managers)`
`should return a list of added beneficiaries by previous managers (managers in a different community)`

Reply:
Ok, I wrote the test about the deleted account. But, the service `getManager` returns just the active managers. Do I need to change the service to return all the managers and the count of beneficiaries added? even the managers that were removed?

Reply:
ahhh nice catch :thinking: 
And that requires changes on my last PR on the app too :thinking: 
Quick though, what if we add a parameter to filter only active, or only inactive, or both?
Something like `/community/{id}/managers/{query}` and here, with a `filterByActive`, if true, returns only active, if false, return only inactive, if undefined returns both.

This way, it works for the public community data (all results) and manager management in-app, filtered by active/inactive. I'm trying to replace an old endpoint https://github.com/impactMarket/mobile-app/pull/225/files#diff-dae814b85d64f1d8e5f9f7a60fb16d881f0ef980b46aae13f50b7318f427ca1bL69.

How does that sound?

Reply:
sounds great, I added this filter!